<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditBookDialog</class>
 <widget class="QWidget" name="EditBookDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="titleLineEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>78</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comfortaa</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="authorLineEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>118</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comfortaa</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="yearLineEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>158</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comfortaa</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>220</y>
     <width>51</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comfortaa</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>save</string>
   </property>
  </widget>
  <widget class="QComboBox" name="stateComboBox">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>190</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comfortaa</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>Avaliable</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Cheked Out</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>70</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comfortaa</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>title</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>110</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comfortaa</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>author</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>150</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comfortaa</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>year</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>41</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comfortaa</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Book Info</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
